src/Cameras/Inputs/arcRotateCameraMouseWheelInput.ts

```diff
@@ -24,6 +27,30 @@ export class ArcRotateCameraMouseWheelInput implements ICameraInput<ArcRotateCam
     @serialize()
     public wheelPrecision = 3.0;
 
+    /**
+     * Gets or Set the ratio that slows down the zooming as the camera radius gets
+     * smaller (i.e. as the camera gets closer to your object).  It is nice if the
+     * mouse wheel steps get smaller so you can be more precise about camera position
+     * as you get close to your object.  If set to non-zero value this causes
+     * wheelPrecision to be computed automatically using the formula
+     * "1 / camera.radius * wheelPrecisionEaseInRatio;".  Note that wheelPrecision
+     * is used like this: "wheelDelta / (this.wheelPrecision * 40);" which means
+     * the larger the wheelPrecision the smaller the zoom amount.  So by inverting
+     * the camera.radius we are getting smaller wheelPrecision amounts the further you
+     * get away from the object, which results in larger zooming steps.  A
+     * wheelPrecisionEaseInRatio of about 300 seems to work nicely in most cases, but
+     * it does depend on the overall size of your scene.
+     */
+     @serialize()
+     public wheelPrecisionEaseInRatio = 0;

```

Comment:
what is the difference with wheelDeltaPercentage ?

Adding @sebavan  and @PolygonalSun as your solution could be better

Reply:
You're right, wheelDeltaPercentage  works, so I removed wheelPrecisionEaseInRatio